website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Integrate with Apple Business Manager
+sidebar_label: Apple Business Manager
+support_level: authentik
+tags: [integration, apple, ssf, backchannel]
+authentik_version: "2025.2.0"
+authentik_enterprise: true
+authentik_preview: true
+---
+
+## What is Apple Business Manager?
+
+> Apple Business Manager is a web-based portal for IT administrators, managers, and procurement professionals to manage devices, and automate device enrollment.
+>
+> Organizations using Apple Business Essentials can allow their users to authenticate into their Apple devices using their IdP credentials, typically their company email addresses.
+>
+> -- [Apple Business Manager](https://www.apple.com/business/)
+
+:::info Apple Device Management Platforms
+
+Apple repackages their device management platform into three brands to cater to different audiences:
+
+- Apple Business Manager: Large organizations
+- Apple Business Essentials: Small businesses
+- Apple School Manager: Educational institutions
+
+While this integration guide focuses on Business Manager, the instructions are applicable to all three platforms with minor changes to the terminology.
+
+:::
+
+## Authentication Flow
+
+This sequence diagram shows a high-level flow between the user's MacBook, authentik, and Apple.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Apple
+
+    User->>Apple: User sign-in via iCloud
+    Note over Apple: 🔍 Email domain is federated
+    Apple-->>authentik: Redirect to authentik
+    Note over authentik: ✅ Authenticate
+    authentik-->>Apple: Identity verified
+    Apple-->>User: Device enrolled!
+
+```
+
+In short, Apple Business Manager recognizes the email domain
+as a federated identity provider controlled by authentik. When a user signs in with their email address, Apple redirects them to authentik for authentication. Once authenticated, Apple enrolls the user's device and grants access to Apple services.
+
+---
```

Comment:
We don't use HRs anywhere else in the docs, so let's remove this one and then have a chat about where we would want to use them, consistently across all docs.